cypress/support/config/services.js

```diff
@@ -1194,10 +1194,7 @@ const genServices = appEnv => ({
       },
       liveRadio: { path: undefined, smoke: false },
       mediaAssetPage: {
-        path:
-          isLive(appEnv) || isTest(appEnv)
-            ? undefined
-            : '/thai/international-49514085',
+        path: isLive(appEnv) ? undefined : '/thai/23122810',
```

Comment:
This page currently 404s on Test `https://www.test.bbc.com/thai/23122810`
```suggestion
        path: isLive(appEnv) || isTest(appEnv) ? undefined : '/thai/23122810',
```

Reply:
Oops good catch - mozart not routing Thai MAPs yet